docs/data-sources/application.md

```diff
@@ -3,7 +3,7 @@
 page_title: "indykite_application Data Source - IndyKite"
 subcategory: ""
 description: |-
-
+  
```

Comment:
Seems you didn't run `pre-commit` at the end

Reply:
indeed :(